aten/src/ATen/cpu/vec256/vec256_base.h

```diff
@@ -43,12 +43,13 @@ namespace at {
 namespace vec256 {
 // See Note [Acceptable use of anonymous namespace in header]
 namespace {
-// at::Half should be treated as floating point
+// at::Half and at::BFloat16 should be treated as floating point
 template <typename T>
 struct is_floating_point:
     std::integral_constant<bool,
       std::is_floating_point<T>::value ||
-      std::is_same<T, at::Half>::value> {
+      std::is_same<T, at::Half>::value ||
+      std::is_same<T, at::BFloat16>::value> {
```

Comment:
Thanks for adding this.

Reply:
Can it be put into separate PR (as it have nothing to do with vectorization goal)